learning-fun/brightspot/src/brightspot/example/learning_fun/LearningFunEndpoint.ts

```diff
@@ -0,0 +1,47 @@
+import Class from 'brightspot-types/java/lang/Class'
+import ContentDeliveryApiEndpoint from 'brightspot-types/com/psddev/graphql/cda/ContentDeliveryApiEndpoint'
+import ContentDeliveryEntryPointField from 'brightspot-types/com/psddev/graphql/cda/ContentDeliveryEntryPointField'
+import GraphQLApiAccessOption from 'brightspot-types/com/psddev/graphql/GraphQLApiAccessOption'
+import GraphQLApiAccessOptionImplicit from 'brightspot-types/com/psddev/graphql/GraphQLApiAccessOptionImplicit'
+import GraphQLCorsConfiguration from 'brightspot-types/com/psddev/graphql/GraphQLCorsConfiguration'
+import JavaClass from 'brightspot-types/JavaClass'
+import JavaSet from 'brightspot-types/java/util/Set'
+import List from 'brightspot-types/java/util/List'
+import Singleton from 'brightspot-types/com/psddev/dari/db/Singleton'
+
+import PageViewModel from './PageViewModel'
+import PagesViewModel from './PagesViewModel'
+
+export default class LearningFunEndpoint extends JavaClass(
+  'brightspot.example.learning_fun.LearningFunEndpoint',
+  ContentDeliveryApiEndpoint,
+  Singleton
+) {
+  getPaths(): JavaSet<string> {
+    return ['/graphql/delivery/pages'] as unknown as JavaSet<string>
+  }
+
+  [`getQueryEntryFields()`](): List<ContentDeliveryEntryPointField> {
+    const arr1 = [PageViewModel.class as Class<PageViewModel>].map(
+      (c) => new ContentDeliveryEntryPointField(c)
+    )
+    const arr2 = [PagesViewModel.class as Class<PagesViewModel>].map(
+      (c) => new ContentDeliveryEntryPointField(c)
+    )
+
+    const finalArray = arr1.concat(
+      arr2
+    ) as unknown as List<ContentDeliveryEntryPointField>
+
+    return finalArray
```

Comment:
Significantly cleaner impl.
```suggestion
    return [
      PageViewModel.class as Class<PageViewModel>,
      PagesViewModel.class as Class<PagesViewModel>,
    ].map(
      (c) => new ContentDeliveryEntryPointField(c),
    ) as unknown as List<ContentDeliveryEntryPointField>
```